code/__DEFINES/inventory.dm

```diff
@@ -3,7 +3,7 @@
 //ITEM INVENTORY WEIGHT, FOR w_class
 #define WEIGHT_CLASS_TINY     1 //Usually items smaller then a human hand, ex: Playing Cards, Lighter, Scalpel, Coins/Money
 #define WEIGHT_CLASS_SMALL    2 //Pockets can hold small and tiny items, ex: Flashlight, Multitool, Grenades, GPS Device
-#define WEIGHT_CLASS_NORMAL   3 //Standard backpacks can carry tiny, small & normal items, ex: Fire extinguisher, Stunbaton, Gas Mask, Metal Sheets
+#define WEIGHT_CLASS_NORMAL   3 //Standard backpacks can carry tiny, small & normal items, ex: Fire extinguisher, stun baton, Gas Mask, Metal Sheets
```

Comment:
The rest of these are capitalised

Reply:
But should it be Stun Baton or Stun baton? :thinking:

Reply:
Why is there both Fire extinguisher and Gas Mask in there, deary me

Reply:
sTuN baTOn